### PR TITLE
Docs: Akka Persistence: sbt/Maven instead of Scala/Java

### PR DESCRIPTION
**Pull request by sebastianharko**
sbt/Maven instead of Scala/Java

### CODE REVIEW

**Review by ktoso — COMMENTED** [review]
Thanks! We started adding gradle too since the new website, would you want to add that to in the same PR perhaps? 

Example: http://akka.io/docs/

**Comment by sebastianharko** [comment]
@ktoso yes ! I'll add gradle too !

**Comment by sebastianharko** [comment]
@ktoso done! please review when u have time :-)

**Review by ktoso — APPROVED** [review]
Great, thanks!

**Comment by sebastianharko** [comment]
@ktoso don't merge just yet, I have one more small fix to add here

**Comment by sebastianharko** [comment]
@ktoso I also fixed the 'dependencies' section under the "Plugin TCK" section.

But reading from the docs "The TCK is usable from Java as well as Scala projects. For Scala you need to include the akka-persistence-tck dependency:" .... This sounds like if you're on Java you don't need the dependency ? Is this true ? If not, this sentence can be removed...

File: akka-docs/src/main/paradox/scala/persistence.md [hunk 1]
```diff
@@ -1117,17 +1132,31 @@ In order to help developers build correct and high quality storage plugins, we p
 
 The TCK is usable from Java as well as Scala projects. For @scala[Scala]@java[Java] you need to include the akka-persistence-tck dependency:
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/persistence.md]: Notice that the note is "smart" on the Java page it'll say Java and for the Scala one it'll say Scala :) 
`@scala[Scala]@java[Java]` cc @sebastianharko  :)

**Review by ktoso — APPROVED** [review]
LGTM, and i clarified that sentence a bit

**Comment by ktoso** [comment]
Thanks!

**Pull request merged**